Meadowlark-js/package.json

```diff
@@ -51,6 +51,7 @@
     "test:integration": "cross-env NODE_OPTIONS=--max-old-space-size=6144 LOG_LEVEL=warn jest --projects backends/meadowlark-mongodb-backend/test/config/integration backends/meadowlark-opensearch-backend/test/config/integration backends/meadowlark-postgresql-backend/test/config/integration --runInBand",
```

Comment:
We should include the Elasticsearch tests here as well

Reply:
Done